lib/protocol/protocol.js

```diff
@@ -7,12 +7,17 @@ import {
 } from './errors';
 import { METHOD_MAP, NO_SESSION_ID_COMMANDS } from './routes';
 import B from 'bluebird';
-import BaseDriver from '../basedriver/driver';
 import {
   MJSONWP_ELEMENT_KEY, W3C_ELEMENT_KEY, formatResponseValue, formatStatus,
 } from './helpers';
 import SESSIONS_CACHE from './sessions-cache';
 
+
+const PROTOCOLS = {
```

Comment:
I'd rather set the corresponding values from BaseDriver here. There is no need to duplicate these strings

Reply:
i thought that's what I'm doing. these strings are not being duplicated, they are being moved (though i am keeping the alias in BaseDriver so as not to make this a breaking change for dependents, similar to how i explained for determineProtocol above)